zstacklib/zstacklib/utils/fileUploader.py

```
@@ -0,0 +1,289 @@
+def get_boundary(entity):
+    # type: (Entity) -> str
+
+    ib = ""
+    if 'boundary' in entity.content_type.params:
+        # http://tools.ietf.org/html/rfc2046#section-5.1.1
+        # "The grammar for parameters on the Content-type field is such that it
+        # is often necessary to enclose the boundary parameter values in quotes
+        # on the Content-type line"
+        ib = entity.content_type.params['boundary'].strip('"')
+
+    if not re.match("^[ -~]{0,200}[!-~]$", ib):
```

Ruff (0.13.1)

`re` may be undefined, or defined from star imports

(F405)